src/app/my-blog/components/layout/layout.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject, takeUntil } from 'rxjs';
+import { IPost } from '@shared/interfaces/blog-item.interface';
+import { BlogService } from '@shared/services/blog.service';
+
+@Component({
+  selector: 'app-layout',
+  templateUrl: './layout.component.html',
+  styleUrls: ['./layout.component.sass'],
+})
+export class LayoutComponent implements OnInit, OnDestroy {
```

Comment:
create post-list and use it inside layout

Reply:
fixed